src/test/mod.rs

```diff
@@ -5,10 +5,9 @@
 //!
 //! The actual tests live with their respective modules in a sub-module called
 //! `interop`. Since they require additional software packages to be available
-//! and can be expensive, they are only run if the `"interop"` feature is
-//! given to `cargo test`.
+//! and can be expensive, they are tagged as ignored by default.
 //!
-#![cfg(all(test, feature = "interop"))]
+#![cfg(all(test, feature = "bytes", feature = "std"))]
```

Comment:
The line removed from `Cargo.toml` grouped the `bytes` and `ring` features into an `interop` feature yet here you replace `interop` with `bytes` and `std` - is that correct?

Reply:
I believe the code in this module only relies on those two features.